bokehjs/test/unit/models/renderers/glyph_renderer.ts

```diff
@@ -41,3 +45,37 @@ describe("GlyphRenderer", () => {
     })
   })
 })
+
+describe("GlyphRendererView", async () => {
+  // Basic case with no all glyphs going to their defaults
+
+  const basic_circle = new Circle({fill_color: "red"})
+  const glyph_renderer = mkrenderer(basic_circle)
+  const view = await build_view(
+    glyph_renderer,
+    {parent: await build_view(new Plot())},
+  )
+  const {glyph, selection_glyph, nonselection_glyph, hover_glyph, muted_glyph, decimated_glyph} = view
+
+  it("should have default selection_glyph equal to main glyph", () => {
+    expect(selection_glyph.model.attributes).to.be.equal(glyph.model.attributes)
+  })
+
+  it("should have default nonselection_glyph with 0.2 alpha", () => {
+    expect(nonselection_glyph.model.attributes.fill_alpha).to.be.equal({value: 0.2})
+  })
+
+  it("should have default hover_glyph model equal to main glyph model", () => {
+    expect(hover_glyph?.model.attributes).to.be.equal(glyph.model.attributes)
+  })
+
+  it("should have default muted_glyph with 0.2 alpha", () => {
+    expect(muted_glyph.model.attributes.fill_alpha).to.be.equal({value: 0.2})
+  })
+
+  it("should have default decimated_glyph with 0.3 line alpha and color grey", () => {
+    expect(decimated_glyph.model.attributes.line_alpha).to.be.equal({value: 0.3})
+    expect(decimated_glyph.model.attributes.line_color).to.be.equal({value: "grey"})
```

Comment:
Use `attributes` here is an unnecessary and inefficient practice (it creates a copy of all attributes and their values). It's sufficient to use `model.line_color`, etc.

Reply:
@mattpap this throws a TypeScript error `Property 'line_color' does not exist on type 'Glyph'`. I created a local interface to get around this. It technically tests fine, but can you confirm it's an acceptable practice before I commit?

Additional imports:
```
import {Attrs} from "@bokehjs/core/types"
import {Glyph} from "@bokehjs/models/glyphs/glyph"
```
New Interface:
```
  interface GlyphWithVisuals extends Glyph {
    fill_alpha: Attrs
    line_alpha: Attrs
    line_color: Attrs
  }
```
Implementation:
```
expect((decimated_glyph.model as GlyphWithVisuals).line_alpha).to.be.equal({value: 0.3})
```

Reply:
Sadly `GlyphRenderer(View)` don't convey enough type information, but that can be locally rectified with a cast as follows:
```ts
expect((nonselection_glyph.model as Circle).fill_alpha).to.be.equal({value: 0.2})
```

Reply:
I like that much better, thank you @mattpap. This is included this in afb1a89